src/test/java/org/jenkinsci/plugins/scriptler/ScriptlerManagementTest.java

```diff
@@ -0,0 +1,46 @@
+package org.jenkinsci.plugins.scriptler;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+import hudson.markup.MarkupFormatter;
+import hudson.markup.RawHtmlMarkupFormatter;
+import java.io.IOException;
+import java.io.Writer;
+import org.junit.jupiter.api.Test;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.jvnet.hudson.test.junit.jupiter.WithJenkins;
+
+@WithJenkins
+class ScriptlerManagementTest {
+
+    @Test
+    void markupFormatter(@SuppressWarnings("unused") JenkinsRule r) throws IOException {
+        ScriptlerManagement management = new ScriptlerManagement();
+
+        // save text
+        String text = management.getMarkupFormatter().translate("Save text");
+        assertEquals("Save text", text);
+
+        // dangerous text with global formatter
+        text = management.getMarkupFormatter().translate("<script>alert('PWND!')</script>");
+        assertEquals("&lt;script&gt;alert(&#039;PWND!&#039;)&lt;/script&gt;", text);
```

Comment:
This appears to assume a formatter, which could break if Jenkins were ever to change the default formatter. Can we explicitly set one before testing any formatting?

Reply:
Since the goal of this PR is to rely on the global formatter it only makes sense to test the default that is being used. If the default would change we would be made aware of it by this test failing which in turn allows us to address a regression / breaking change.

Reply:
Fair enough. Not quite sure if it completely agree, but it's also not a deal breaker for me.